packages/mdc-chips/chip-set/foundation.js

```diff
@@ -82,24 +81,25 @@ class MDCChipSetFoundation extends MDCFoundation {
    */
   handleChipInteraction_(evt) {
     const {chip} = evt.detail;
+    const chipFoundation = chip.foundation;
```

Comment:
Should we document this requirement in the README?

Wrapper libraries will probably need to know about it.